src/commonMain/kotlin/com/eiknat/lemmyclient/api/user/model/User.kt

```diff
@@ -22,15 +22,15 @@ data class User(
     val showNsfw: Boolean,
     val theme: String,
     @SerialName("default_sort_type")
-    val defaultSortType: SortType,
+    val defaultSortType: Int, // should these return SortType/ListingType?
```

Comment:
SortType was fine I think?

Reply:
it comes back as an integer, same with listing type. i was gonna ask dessalines about it since it seems to be the only place that does that

Reply:
https://github.com/LemmyNet/lemmy/issues/1136